src/fides/api/models/detection_discovery.py

```diff
@@ -348,6 +338,41 @@ def mark_as_addition(
                 parent_resource.add_child_diff_status(DiffStatus.ADDITION)
 
 
+class MonitorExecution(Base):
+    """
+    Monitor execution record used for data detection and discovery.
+
+    Each monitor execution references `MonitorConfig`, which provide it with underlying
+    configuration details used in connecting to the external data store.
+    """
+
+    id = Column(String, primary_key=True)
+    monitor_config_key = Column(
+        String,
+        ForeignKey(MonitorConfig.key),
+        nullable=False,
+        index=True,
+    )
+    status = Column(String, nullable=True)
+    started = Column(
+        DateTime(timezone=True), nullable=True, default=datetime.now(timezone.utc)
+    )
+    completed = Column(DateTime(timezone=True), nullable=True)
+    classification_instances = Column(
+        ARRAY(String),
+        index=False,
+        unique=False,
+        nullable=False,
+        default=list,
+    )
+    created_at = Column(
+        DateTime(timezone=True), nullable=False, default=datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), nullable=False, default=datetime.now(timezone.utc)
+    )
```

Comment:
pretty sure these are inferred from the `Base` model class, a bit cleaner to not re-define here unless it's needed for some reason?

Reply:
Swapped to how we did it in the most recent migration here: https://github.com/ethyca/fides/pull/5704/commits/af165b5cd6cdc558506a12a03d4081b274980614